libs/api/mocks/src/domains/license-service/store.ts

```diff
@@ -16,8 +16,13 @@ export const store = createStore(() => {
   const getLicenses = () => {
     if (!licenses.length) {
       licenses = genericUserLicenses(
-        ['AdrLicense', 'FirearmLicense', 'DriversLicense', 'MachineLicense'] ??
-          [],
+        [
+          'DriversLicense',
+          'FirearmLicense',
+          'AdrLicense',
+          'MachineLicense',
+          'DisabilityLicense',
+        ] ?? [],
```

Comment:
```suggestion
        ],
```

this will never happen since an array is always truthy